data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -0,0 +1,296 @@
+privacy_notices:
+- name: Essential
+  notice_key: essential
+  id: pri_51c85583-40ec-4257-95ef-db3eea860ess
```

Comment:
ah it was a modification of the FidesBase.generate_uuid, except I replaced the last three characters with the first three characters of the notice_key.  It would be nice to just rely on "notice_key" but these are not at all guaranteed to be unique!